aries-site/src/pages/templates/page-layouts.mdx

```diff
@@ -92,7 +92,7 @@ To make the following functionality easy to implement, Grommet provides
   <PageAnatomy />
 </Example>
 
-### Page Kinds
+### Page kinds
```

Comment:
@taysea seems odd to use "kinds" here
Is "types" an accurate substitute, or does that imply something different?

Reply:
We use the term "kinds" because `kind` is the grommet property applied.